config/deploy/web-hpa.yaml.erb

```diff
@@ -0,0 +1,38 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: web-hpa
+  labels:
+    app: web
+    environment: <%= environment %>
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: web
+  minReplicas: <%= environment == 'production' ? 3 : 2 %>
+  maxReplicas: <%= environment == 'production' ? 10 : 5 %>
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        type: Utilization
+        averageUtilization: 75
+  behavior:
+    scaleDown:
+      stabilizationWindowSeconds: 300
+      policies:
+      - type: Percent
+        value: 50
+        periodSeconds: 60
+    scaleUp:
+      stabilizationWindowSeconds: 60
+      policies:
+      - type: Percent
+        value: 100
```

Comment:
Are our web pods hitting 100% CPU? Can you check with Datadog to review what the ideal CPU usage percentage would be for scaling up?

Reply:
Our web pods are not hitting 100+% often, but it has happened a few times in the past month 👇

<img width="1285" height="559" alt="image" src="https://github.com/user-attachments/assets/a9123bc2-45d6-4954-8fcf-9cd15e6fb8ba" />